index.html

```diff
@@ -16,9 +16,8 @@
 <body>
 	<div class="top-section">
 		<h1>Tic - Tac - Toe</h1>
-		<h2>Current Turn: <span id="currentplayer">Player 1</span></h2>
-		<h2 class="display-results"></h2>
```

Comment:
I reduced this to a single h2, as the second one was making the ui jumpy. Also the less html the better.